libnmstate/error.py

```diff
@@ -79,6 +79,17 @@ class NmstateVerificationError(NmstateError):
     pass
 
 
+class NmstateKernelIntergerRounded(NmstateVerificationError):
```

Comment:
I think this error name is missing the "Error", In addition, there is a typo in the "Integer". It should be `NmstateKernelIntegerRounded`